images/macos/provision/core/node.sh

```diff
@@ -40,8 +35,6 @@ echo Installing yarn...
 curl -o- -L https://yarnpkg.com/install.sh | bash
 
 if is_Less_BigSur; then
```

Comment:
what's wrong with Big Sur and node-gyp?

Reply:
We wouldn't like to bring useless module to future images and can't remove it from existing ones